src/sdam/server_description.ts

```diff
@@ -135,6 +135,16 @@ class ServerDescription {
     return WRITABLE_SERVER_TYPES.has(this.type);
   }
 
+  get host() {
+    const chopLength = `:${this.port}`.length;
+    return this.address.slice(0, -chopLength);
+  }
+
+  get port() {
+    const addressParts = this.address.split(':');
+    return Number.parseInt(addressParts[addressParts.length - 1], 10);
```

Comment:
I probably would have done `return Number.parseInt(this.address.split(':').pop(), 10);` here for brevity. Just wondering if you chose this slightly more verbose approach for performance reasons (I'm assuming a pop would be slightly slower than the way you've done it) or if it was just a stylistic choice.

Reply:
I'm just a bad javascript programmer who forgot pop() existed 😂 I can totally change it, that does read nicer